javascript.md

```diff
@@ -0,0 +1,793 @@
+# JavaScript Guide
+
+This style guide aims to provide the ground rules for CFPB's JavaScript code, such that it's highly readable and consistent across different developers on a team. 
+
+
+## Table of Contents
+
+1. [Linting](#linting)
+1. [Modules](#modules)
+1. [Strict Mode](#strict-mode)
+1. [Spacing](#spacing)
+1. [Naming](#naming)
+1. [Semicolons](#semicolons)
+1. [Strings](#strings)
+1. [Variable Declaration](#variable-declaration)
+1. [Conditionals](#conditionals)
+1. [Equality](#equality)
+1. [Ternary Operators](#ternary-operators)
+1. [Functions](#functions)
+1. [Prototypes](#prototypes)
+1. [Object Literals](#object-literals)
+1. [Array Literals](#array-literals)
+1. [Regular Expressions](#regular-expressions)
+1. [`console` Statements](#console-statements)
+1. [Comments](#comments)
+1. [Variable Naming](#variable-naming)
+1. [Polyfills](#polyfills)
+1. [Everyday Tricks](#everyday-tricks)
+1. [Credits](#credits)
+
+## Linting
+
+Lint your code using ESLint. [CFPB-tailored ESLint options](https://github.com/cfpb/front-end/js-standards/.eslintrc) exist.
+
+## Modules
+
+This style guide assumes you're using a module system such as [CommonJS][1], [AMD][2], [ES6 Modules][3], or any other kind of module system. Modules systems provide individual scoping, avoid leaks to the `global` object, and improve code base organization by **automating dependency graph generation**, instead of having to resort to manually creating tens of `<script>` tags.
```

Comment:
Yes, I would think so. It may even read better as "...resort to manually creating multiple `<script>` tags."